pandas/tests/api/test_protocol.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+import math
```

Comment:
api tests are really just for namespace checking.

real tests should be in pandas/tests/test_exchange.py or similar